charts/partials/templates/_proxy.tpl

```diff
@@ -1,5 +1,9 @@
 {{ define "partials.proxy" -}}
 env:
+{{ if .Values.global.proxy.requireIdentityOnInboundPorts -}}
+- name: LINKERD2_PROXY_INBOUND_PORTS_REQUIRE_IDENTITY
+  value: {{.Values.global.proxy.requireIdentityOnInboundPorts}}
```

Comment:
One problem I did find afterwards is that this value requires to be wrapped in quotes, otherwise if a single port is used here it's interpreted as an int value and it breaks the injector because a string is expected

Reply:
yeah very good catch. Thanks!